### PR TITLE
EZP-22700 Document making backups

### PR DESCRIPTION
**Pull request by MagdalenaZuba**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | https://jira.ez.no/browse/EZP-22700
| Versions      | all

Document making backups

### CODE REVIEW

File: docs/migrating/backup.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Backup 
+
+You should always make sure that your solution is properly backed up. The following example shows you how to do this on a Linux/UNIX based system where eZ Platform is using a MySQL database called "example". You should close the system before making a backup or an upgrade.
```
Comment (DominikaK) [inline docs/migrating/backup.md]: `Linux-UNIX-based`
Not sure about `close the system`, sounds like closing the OS, is that what is meant? Or shut down Platform if it's running?

File: docs/migrating/backup.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+!!! note "Externally stored assets"
+
+    If you store assets in any external service or localization, you should back them up before proceeding.
+
+1\. Navigate into the eZ Platform directory:
+ 
+```
+$ cd /path/to/ezplatform
```
Comment (DominikaK) [inline docs/migrating/backup.md]: We don't use the `$` before bash commands elsewhere, let's remove it here.
Also applies below

File: docs/migrating/backup.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+$ var/<var_dir>/cache/*
+```
+
+4\. Create a dump of the database:
+ 
+```
+$ mysqldump -u root --add-drop-table example > db_backup.sql
+```
+
+5\. In parent directory create a tape archive of the files (including the DB dump) using the "tar" command:
```
Comment (DominikaK) [inline docs/migrating/backup.md]: Not sure if "tape archive" is used as a full term, I think "tar" is more common and less confusing.

File: docs/migrating/backup.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+```
+$ tar cfz backup_of_ezplatform.tar.gz ezplatform
+```
+
+ At this point, the file "backup_of_ezplatform.tar.gz" should contain a backup of DB and files.
+
+## Consistency checks
+
+You can check if your installation is in a consistent state by running below checks from the "Upgrade check" section of the "Setup".
```
Comment (DominikaK) [inline docs/migrating/backup.md]: AFAIK we don't have an "Upgrade check" section

File: docs/migrating/backup.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+$ mysqldump -u root --add-drop-table example > db_backup.sql
+```
+
+5\. In parent directory create a tape archive of the files (including the DB dump) using the "tar" command:
+
+```
+$ tar cfz backup_of_ezplatform.tar.gz ezplatform
+```
+
+ At this point, the file "backup_of_ezplatform.tar.gz" should contain a backup of DB and files.
```
Comment (DominikaK) [inline docs/migrating/backup.md]: File name in backticks + uneccessary space at line start

File: docs/migrating/backup.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+This tool checks for any alterations in files that came with the eZ Platform installation. You should back up those files before the system upgrade because they will be overwritten by the new versions. After upgrade you can merge changed files into the new version.
+
+### Database consistency check
+
+This feature checks if the database is consistent with schema that came with currently running eZ Platform distribution. The system will suggest the necessary SQL statements to ran in order to bring the database into a consistent state. Make sure that the database is backed up and run the suggested SQL statements before upgrading.
+
+!!! note "eZ Network"
+
+    If your installation does not have the eZ Network installed, you will get an error message about database inconsistency. This is to be expected and the error message will disappear as soon as the eZ Network is installed.
```
Comment (DominikaK) [inline docs/migrating/backup.md]: Is eZ Network a thing still, now in "Platform era"?

File: docs/migrating/backup.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+You can check if your installation is in a consistent state by running below checks from the "Upgrade check" section of the "Setup".
+ 
+### File consistency check
+
+This tool checks for any alterations in files that came with the eZ Platform installation. You should back up those files before the system upgrade because they will be overwritten by the new versions. After upgrade you can merge changed files into the new version.
+
+### Database consistency check
+
+This feature checks if the database is consistent with schema that came with currently running eZ Platform distribution. The system will suggest the necessary SQL statements to ran in order to bring the database into a consistent state. Make sure that the database is backed up and run the suggested SQL statements before upgrading.
```
Comment (DominikaK) [inline docs/migrating/backup.md]: `consistent with the schema that came with the current eZ Platform distribution`
`the necessary SQL statements to rUn`

**Comment by MagdalenaZuba** [comment]
ping @Philippe for review

File: docs/migrating/backup.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+    If you store assets in any external service or localization, you should back them up before proceeding.
+
+1\. Navigate into the eZ Platform directory:
+ 
+```
+$ cd /path/to/ezplatform
+```
+
+2\. Unclusterize or backup the NFS folder.
```
Comment (DominikaK) [inline docs/migrating/backup.md]: Is it possible to explain in more detail how to do it?

**Pull request merged**